core/src/main/java/org/elasticsearch/search/suggest/completion/CompletionSuggestionBuilder.java

```diff
@@ -125,6 +122,26 @@ private CompletionSuggestionBuilder(String fieldname, CompletionSuggestionBuilde
     }
 
     /**
+     * Read from a stream.
+     */
+    public CompletionSuggestionBuilder(StreamInput in) throws IOException {
+        super(in);
+        payloadFields = new ArrayList<>();
```

Comment:
like if these were still builders :) not really we may prefer lists when we have methods that allow to add rather than set, but this is not the case.